tests/PHPSA/ApplicationTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Tests\PHPSA;
+
+use PHPSA\Application;
+
+class ApplicationTest extends TestCase
+{
+    /**
+     * @covers \PHPSA\Application::__construct
+     */
+    public function testConstructor()
+    {
+        $application = new Application();
+
+        $this->assertInstanceOf('\PHPSA\Application', $application);
+        $this->assertInstanceOf('\Symfony\Component\Console\Application', $application);
```

Comment:
I don't know what @ovr and @ddmler think about that, but I don't really see the value in this kind of assertions.
Tests should reflect the behavior of the system, how it reacts given some input data, … Just testing that a class can be instantiated does not seem that useful to me.